cluster/kubemark/gce/config-default.sh

```diff
@@ -22,93 +22,20 @@
 # shellcheck disable=SC2034 # Variables sourced in other scripts.
 
 source "${KUBE_ROOT}/cluster/gce/config-common.sh"
+NUM_NODES="${KUBEMARK_NUM_NODES:-}"
```

Comment:
What about:
NUM_WINDOWS_NODES
MASTER_SIZE
MASTER_DISK_SIZE
MASTER_ROOT_DISK_SIZE
?

Those are things that we allow overwriting differently for kubemark (similarly as we allow overwriting for regular gce clusters) and I think it's actually useful ability to have

Reply:
Never mind - I see that is happening below.